app/controllers/application_controller.rb

```diff
@@ -1,5 +1,15 @@
 class ApplicationController < ActionController::Base
-  def current_user
-    User.first
+  protect_from_forgery with: :exception
+
+  before_action :update_allowed_parameters, if: :devise_controller?
+
+  protected
+
+  def update_allowed_parameters
+    devise_parameter_sanitizer.permit(:sign_up) | u |
+      u.permit(:name, :photo, :bio, :email, :password, :password_confirmation)
+
+    devise_parameter_sanitizer.permit(:account_update) | u |
+      u.permit(:name, :photo, :bio, :email, :password, :password_confirmation)
   end
 end
```

Comment:
- [ ] there is an error when a user tries to sign in, kindly fix this
![Screenshot from 2022-08-27 21-28-59](https://user-images.githubusercontent.com/66097371/187047013-29c492df-2911-4726-858b-4aaa1c49eb1a.png)
hint: you can change your method to :point_down: 
```
def update_allowed_parameters
    devise_parameter_sanitizer.permit(:sign_up) { |u| u.permit(:name, :email, :password) }
    devise_parameter_sanitizer.permit(:account_update) do |u|
    u.permit(:name, :email, :password, :current_password)
    end
  end
end
```